setup.cfg

```diff
@@ -0,0 +1,6 @@
+[isort]
```

Comment:
isort in setup.cfg? i've never seen that, but i'm also not doing much CI setups.